### DISCUSSION TITLE
add on one chrome extension

### DISCUSSION THREAD

**Discussion by xuhongjian** [cefsharp/CefSharp#4548] [Q&A]
i have a chrome extension,in chrome is running perfect,but in cefsharp there have an error  非常抱歉， gj 需要开启浏览器的Javascript执行功能方可运行. 请开启后再继续.  although  add code  var settings = new BrowserSettings();
            settings.Javascript = CefState.Enabled;
            browser.BrowserSettings = settings;  the error still exsist, how to solve it.  extension code is here  https://t.wss.ink/f/bq7ubkpefrj

**Comment by amaitland** [comment]
English please.

You need to use the **Chrome Runtime** to use extensions https://github.com/cefsharp/CefSharp/discussions/4123

There's no programmatic support for loading extensions in the `Chrome Runtime`.
